controller/controller.go

```diff
@@ -64,6 +70,31 @@ func main() {
 		name.SetSeed(s)
 	}
 
+	var tokenKey *ecdsa.PublicKey
+	if tk := os.Getenv("ACCESS_TOKEN_KEY"); tk != "" {
```

Comment:
When `ACCESS_TOKEN_KEY` is not set, `tokenKey` remains `nil` which causes a panic if `authorizer.AuthorizeToken` is called. What should the behaviour be in this case? Should the controller crash at boot when this env is not set, or should a key be generated?